### ISSUE TITLE
https support for exporter

### ISSUE THREAD

**Issue by vilhelmprytz** [prometheus-pve/prometheus-pve-exporter#130]
Would be nice if the exporter could run over https (meaning adding options to read certificate and key from specified files).

[Node Exporter](https://github.com/prometheus/node_exporter) allows for the option `--web.config.file=""`, which can be set to the path of a YAML file with contents like this.

```yml
tls_server_config:
  cert_file: /etc/letsencrypt/live/..../fullchain.pem
  key_file: /etc/letsencrypt/live/..../privkey.pem
```

I think there is a Go package that implements support for this (see [this](https://github.com/prometheus/exporter-toolkit/blob/v0.1.0/https/README.md)).

**Comment by znerol** [comment]
True. Regrettably `pve_exporter` is written in Python. I wouldn't reject a rewrite in go (provided that somebody is willing to help with the task and also with longer term project maintenance #84).

**Comment by znerol** [comment]
Resolved via #132 and released in v2.3.0
